### DISCUSSION TITLE
Could we revert to earlier icons for arrangement?

### DISCUSSION THREAD

**Discussion by asos-alexwhite** [structurizr/lite#108] [Ideas]
The alignment icons on the current version of structurizr lite are really bad for usability. After a couple of weeks on the latest version I'm still not sure what they mean. This contrasts with the previous version which was crystal clear about what alignment or distribution was going to happen when clicking them.  I'm particularly fearful of accidentally clicking the wrong 'distribute' icon and messing things up. 

I think this is because the earlier icon design matched the mental model which is used in all the diagram editing programs I've used for years, and these ones don't. 

Please consider reverting to the earlier version of the icons. 

Current icons:

<img width="346" alt="image" src="https://github.com/structurizr/lite/assets/79163907/d143b5b2-f5b1-495c-b90e-86cd94bc72d7">

**Comment by simonbrowndotje** [comment]
The old icons were a licensed product, and had to be removed when the codebase became open source. Sorry!

**Reply by asos-alexwhite** [reply]
Darn it! Well, I can't say anything about that!

Perhaps a change to the distribute icons so that all three lines are the same width? That would help convey that it is distributing things rather than something else (because the thin line means direction of alignment in the other cases, but it doesn't for the distribution). 

Would that be worth considering?

**Reply by simonbrowndotje** [reply]
The icons are open source, so you could open an issue on the repo -> https://github.com/twbs/icons
